content/docs/hooks/use-clipboard.mdx

```diff
@@ -46,6 +47,39 @@ function Example() {
         <Button onClick={onCopy} ml={2}>
           {hasCopied ? 'Copied' : 'Copy'}
         </Button>
+        <Button onClick={() => setValue('New value is there')}>
+          Change value
+        </Button>
+      </Flex>
+      <Editable placeholder='Paste here'>
+        <EditablePreview width='100%' />
+        <EditableInput />
+      </Editable>
+    </>
+  )
+}
+```
+
+Using the `setValue` method to update the value to be copied.
+Remember, this method just changes the current value of hook that will be copied by calling `onCopy` method then.
+So, if we want to copy the new value into clipboard. we need to fire `onCopy` method again.
+
+Note: the value of `useClipboard` hook is not updated immediately after calling `setValue` method because it's a state. So, if we want to copy the new value into clipboard after we call `setValue` method, we should use `useEffect` hook to listen the change and trigger the `onCopy` method with new value.
```

Comment:
Suggestion:

```
Use the `setValue` method to update the value that we want to copy. This method does not update the clipboard, it just changes the current value. So, if we want to copy the new value into clipboard we need to execute the `onCopy` method again.

> Note: the value of `useClipboard` hook is not updated immediately after calling `setValue` method because it's a 
> state variable.
```